### ISSUE TITLE
does not work for me

### ISSUE THREAD

**Issue by Pierre20230** [juergenpabel/OctoPrint-Procastinator#11]
Hello,
Thanks for your work. Great idea which I'd love to use but it does not work for me.
Prusa MK3S, latest version firmware, latest version of octoprint etc,
I can get to the dialog in the time windows correctly and define a start time.
It will start at the right time, heat correctly but just does not print.
Any idea ?
Thanks.

**Comment by juergenpabel** [comment]
Not really, but I haven't tested with 1.5.x (haven't had time to print anything lately), but I'll test that sometime soonish (probably this weekend).

**Comment by Pierre20230** [comment]
Hi Jurgen,

Thank you for quick response and also for your work.
Absolutely no urgency and I’m sure you have other things to do 😉
Best,

Pierre

**Comment by JLGBJ** [comment]
I have the same issue with OctoPrint 1.5.3

**Comment by Ionikekko** [comment]
Dear, with ver 1.6.1 not works.
thanks in adavance for your time.

Roberto
